src/templates/scripts.js

```diff
@@ -230,6 +251,12 @@ const init = () => {
         tagsCheckbox.checked = 'true';
       }
     }
+    if (localStorage.getItem('darkMode') !== null && localStorage.getItem('darkMode') === 'true') {

```

Comment:
Why are you using `localStorage.getItem` here instead of just saying `localStorage.darkMode` like we did for the cfDisplayTags pref?  I don't have a strong preference on which one to use, but we should say consistent.

Reply:
getItem is the preferred standard, i'll go change the cfDisplayTags to follow the pattern.